### ISSUE TITLE
do not use root user

### ISSUE THREAD

**Issue by runningman84** [metalmatze/alertmanager-bot#65]
I think this container would also run just fine if it would run as user nobody. I would advice to add a user directive to the Dockerfile.

**Comment by metalmatze** [comment]
Yes. That's totally right. Would you like to update the documentation?

**Comment by runningman84** [comment]
I can send you a pull request for the Dockerfile. I would also add tini as an entrypoint which is also best practice.
